networkx/algorithms/tests/test_maximum_weight_cycle_packing.py

```diff
@@ -0,0 +1,136 @@
+import networkx as nx
+from networkx.algorithms.approximation.Maximum_weight_cycle_packing_approximation_algorithm import (
+    Maximum_weight_cycle_packing_approximation_algorithm,
```

Comment:
The tests of the approximation algorithm should be in the folder `algorithms/approximation/tests.`